README.md

```diff
@@ -1,10 +1,3 @@
-[![npm][npm]][npm-url]
```

Comment:
Please leave the README as is the template there is not up-to date and will change